back/infolica/views/facture.py

```diff
@@ -130,13 +127,35 @@ def factures_delete_view(request):
     if not Utils.has_permission(request, request.registry.settings['affaire_facture_edition']):
         raise exc.HTTPForbidden()
 
-    id = request.params['id'] if 'id' in request.params else None
+    facture_id = request.params['id'] if 'id' in request.params else None
+
+    # Check if emolument_facture_repartition exists and delete it before
+    results = request.dbsession.query(
+        EmolumentAffaireRepartition
+    ).filter(
+        EmolumentAffaireRepartition.facture_id == facture_id
+    ).all()
+
+    if len(results) > 0:
```

Comment:
Sauf erreur un `if results` devrait suffire (sauf erreur, si pas de résultats, `results = None`)

Reply:
Perso j'ai eu ceci:
![image](https://user-images.githubusercontent.com/7292688/152345481-5cc4285f-10e5-4b44-b14d-31c6ed211809.png)

Je vais adapter par `if results and len(results) > 0:`